src/application/tester.go

```diff
@@ -61,6 +58,24 @@ func (a *Tester) RunAppTests(serviceName string) error {
 	return a.Logger.Log("exo-test", fmt.Sprintf("%d tests failed", numFailed), true)
 }
 
+func (a *Tester) RunServiceTest(serviceName string) error {
+	a.logChannel <- fmt.Sprintf("Testing application %s", a.AppConfig.Name)
+	numFailed := 0
+	if a.InternalServiceConfigs[serviceName].Tests == "" {
+		a.logChannel <- fmt.Sprintf("%s has no tests, skipping", serviceName)
+	} else {
+		if testPassed, err := a.runServiceTests(serviceName, a.InternalServiceConfigs[serviceName]); err != nil {
+			a.logChannel <- fmt.Sprintf("error running '%s' tests:", err)
+		} else if !testPassed {
+			numFailed++
+		}
+	}
+	if numFailed == 0 {
+		return a.Logger.Log("exo-test", "All tests passed", true)
+	}
+	return a.Logger.Log("exo-test", fmt.Sprintf("%d tests failed", numFailed), true)
```

Comment:
I don't think we need some of these log messages. "Testing application" / "All tests passed" / "%d tests failed" aren't really needed since we are only running the tests for a single service.